### PR TITLE
removed major, critical vulnerabilities

### PR DESCRIPTION
**Pull request by akhilesh-singh-Lab49**
reduced major, critical vulnerabilities. Down from 68 to 3.

### CODE REVIEW

File: integrationTest/factory.ts [hunk 1]
```diff
@@ -258,7 +258,7 @@ const getTreasureBalance = async (tokens: Array<TokenId>) => {
 async function main() {
   await setupFactory();
   await testForSinglePair(contractId, tokenC, tokenB);
-  await testForSinglePair(contractId, tokenC, tokenD);
+  // await testForSinglePair(contractId, tokenC, tokenD);
```
Comment (amandeeprehal-lab49) [inline integrationTest/factory.ts:261]: pls remove this change

File: test/pair-test.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { expect } from "chai";
 import { loadFixture } from "@nomicfoundation/hardhat-network-helpers";
 import { ethers, upgrades } from "hardhat";
 import { BigNumber, Overrides, PayableOverrides } from "ethers";
-import Web3 from "web3";
+// import Web3 from "web3";
```
Comment (amandeeprehal-lab49) [inline test/pair-test.ts:7]: why this change?

Comment (akhilesh-singh-Lab49) [reply test/pair-test.ts:7]: Removed as it's not used any where in file. Should I commit it separately?

**Review by amandeeprehal-lab49 — APPROVED** [review]

**Review by amandeeprehal-lab49 — APPROVED** [review]
